buildSrc/src/main/kotlin/configuration/FlavorsEnum.kt

```diff
@@ -7,12 +7,12 @@ enum class FlavorsEnum(
 ) {
     PRODUCTION(
         flavorName = "production",
-        baseUrl = "https://foodish-api.herokuapp.com/",
+        baseUrl = "https://random.imagecdn.app/",
```

Comment:
The old API was not working anymore. Migrated to a new one that generates a random image.